docs/openapi/components/schemas/common/EDDSHP.yml

```
@@ -0,0 +1,397 @@
+$linkedData:
+  term: EDDSHP
+  '@id': https://w3id.org/traceability#EDDSHP
+title: EDD Shapefile
+description: EDD (Early Detection and Distribution) mapping system shapefile.
+type: object
+properties:
+  type:
+    oneOf:
+      - type: array
+        items:
+          type: string
+          enum:
+            - EDDSHP
+      - type: string
+        const: EDDSHP
+  meta:
+    title: Metainformation
+    description: The corresponding EDD Shapefile Metainformation.
+    $ref: ./EDDSHPMeta.yml
+    $linkedData:
+      term: meta
+      '@id': https://w3id.org/traceability#EDDSHPMeta
+  reporter:
+    title: Reporter
+    description: Name of the person, agency, group, organization reporting the data.
```

`and`? `or`? `and/or`?
```suggestion
    description: Name of the person, agency, group, and/or organization reporting the data.
```